lib/ansible/plugins/callback/json.py

```diff
@@ -72,13 +90,37 @@ def v2_playbook_on_stats(self, stats):
             s = stats.summarize(h)
             summary[h] = s
 
+        custom_stats = {}
+        if self._plugin_options.get('show_custom_stats', C.SHOW_CUSTOM_STATS) and stats.custom:  # fallback on constants for inherited plugins missing docs
```

Comment:
you don't need the constant, the config definition above already covers this

Reply:
I had copied that directly from the default callback.  I've updated to use `self.get_option` instead.

Reply:
the default had to allow for '3rd party callback' subclassing and not using the new config framework